packages/backend/src/paymentPointerKey/routes.ts

```diff
@@ -24,15 +38,22 @@ export async function getKeysByPaymentPointerId(
   deps: ServiceDependencies,
   ctx: PaymentPointerContext
 ): Promise<void> {
-  if (!ctx.paymentPointer) {
-    return ctx.throw(404)
-  }
-
-  const keys = await deps.paymentPointerKeyService.getKeysByPaymentPointerId(
-    ctx.paymentPointer.id
-  )
+  if (ctx.paymentPointer) {
+    const keys = await deps.paymentPointerKeyService.getKeysByPaymentPointerId(
+      ctx.paymentPointer.id
+    )
 
-  ctx.body = {
-    keys: keys.map((key) => key.jwk)
+    ctx.body = {
+      keys: keys.map((key) => key.jwk)
+    }
+  } else if (
+    deps.config.paymentPointerUrl ===
+    `https://${ctx.request.host}/${ctx.params.paymentPointerPath}`
```

Comment:
Should this be made into a function we can use across this package? Any function name will also help with the self-commenting of the code IMO

Reply:
I'd like to be able to do this via a `paymentPointerUrl` getter on `AppContext`(?), but I'm struggling to pull it off.